frontend/openchat-agent/src/services/openchatAgent.ts

```diff
@@ -42,138 +42,139 @@ import {
+    type PinChatResponse,
+    type PinMessageResponse,
+    type PublicProfile,
+    type RegisterPollVoteResponse,
+    type RegisterProposalVoteResponse,
+    type RegisterUserResponse,
+    type RemoveMemberResponse,
+    type ResetInviteCodeResponse,
+    type SearchDirectChatResponse,
+    type SearchGroupChatResponse,
+    type SendMessageResponse,
+    type SetBioResponse,
+    type SetUsernameResponse,
+    type StakeNeuronForSubmittingProposalsResponse,
+    type StorageStatus,
+    type SuspendUserResponse,
+    type ThreadPreview,
+    type ThreadPreviewsResponse,
+    type ThreadSyncDetails,
+    type ToggleMuteNotificationResponse,
+    type UnblockUserResponse,
+    type UndeleteMessageResponse,
+    type UnpinChatResponse,
+    type UnpinMessageResponse,
+    type UpdateGroupResponse,
+    type User,
+    type UserCanisterGroupChatSummary,
+    type UserCanisterGroupChatSummaryUpdates,
+    type UserCanisterCommunitySummary,
+    type UserCanisterCommunitySummaryUpdates,
+    type UserLookup,
+    type UsersArgs,
+    type UsersResponse,
+    type UserSummary,
+    type WithdrawCryptocurrencyResponse,
+    type FreezeGroupResponse,
+    type UnfreezeGroupResponse,
+    type UnsuspendUserResponse,
+    type ChatStateFull,
+    type ChatSummary,
+    type UpdatesResult,
+    type DeletedGroupMessageResponse,
+    type DeletedDirectMessageResponse,
+    type ClaimPrizeResponse,
+    type DiamondMembershipDuration,
+    type PayForDiamondMembershipResponse,
+    type AddHotGroupExclusionResponse,
+    type RemoveHotGroupExclusionResponse,
+    type SetCommunityModerationFlagsResponse,
+    type SetGroupUpgradeConcurrencyResponse,
+    type SetUserUpgradeConcurrencyResponse,
+    type UpdateMarketMakerConfigArgs,
+    type UpdateMarketMakerConfigResponse,
+    type ProposalVoteDetails,
+    type SetMessageReminderResponse,
+    type ReferralLeaderboardRange,
+    type ReferralLeaderboardResponse,
+    type InviteUsersResponse,
+    type DeclineInvitationResponse,
+    type AccessGate,
+    type JoinCommunityResponse,
+    type GroupSearchResponse,
+    type ChatIdentifier,
+    type DirectChatIdentifier,
+    type GroupChatIdentifier,
+    type MessageContext,
+    type CommunitySummary,
+    type ExploreCommunitiesResponse,
+    type ChannelIdentifier,
+    type MultiUserChatIdentifier,
+    type CommunityIdentifier,
+    type CommunitySummaryResponse,
+    type UpdatesSuccessResponse,
+    type ConvertToCommunityResponse,
+    type ExploreChannelsResponse,
+    type CommunityInvite,
+    type RegistryValue,
+    type PublicGroupSummaryResponse,
+    type SetDisplayNameResponse,
+    type CreateUserGroupResponse,
+    type UpdateUserGroupResponse,
+    type DeleteUserGroupsResponse,
+    type SetMemberDisplayNameResponse,
+    type UpdatedRules,
+    type FollowThreadResponse,
+    type CandidateProposal,
+    type SubmitProposalResponse,
+    type AccountTransactionResult,
+    type OptionalChatPermissions,
+    Stream,
```

Comment:
We have 2 import blocks from `openchat-shared`, one is all the types and one is everything else. This should be in the block just below this one.